bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -489,29 +511,38 @@ public void create(Composite parent, int style) {
 void browserDispose(Event event) {
 	containingEnvironment.instances.remove(this);
 
-	if (webView_2 != null) webView_2.Release();
 	if (environment2 != null) environment2.Release();
```

Comment:
Is it correct ot release the environment before releasing the actual webview instance? I would expect that this should also be called within the async execution releasing the webview instance.